cs481-ekh/f21-welolo
# Refactor to self hosted

Branch is up as 'refactor_to_self_hosted'.

Verification Steps:
1. Run build.sh (required)
2. Open 2 terminals and navigate to the client and server folders in each terminal
3. Run 'npm start' in both terminals
4. Navigate to 'localhost:3000'
5. Verify a list of merchants is populated and with a name, photo, and selector button
6. Select any merchant via the 'Select Merchant' button on one of the merchant tiles
7. Verify you are redirected to 'localhost:3000/view_menu/merchant_id' and there is a list of items for that merchant that contains a name, description, and image
8. Select an item using the 'Select Item' button on one of the item tiles
9. Verify you are redirected to 'localhost:3000/pay_forward/item_id' and all input boxes are rendered on the screen. Ensure the quantity text box is filled in and not editable.
10. Fill out all form fields with valid information and click submit
11. Verify the payment modal pops up
12. Fill out the payment modal using the test payment credentials and verify the transaction is authorized
13. Repeat the above steps but instead of selecting an item, scroll to the bottom of the page and select 'Skip this step'
